deployment/v2/3_deployContracts.ts

```diff
@@ -301,7 +301,7 @@ async function main() {
         )
     ).data;
 
-    const dataCallProxy = polygonZkEVMBridgeFactory.interface.encodeFunctionData("initialize", [
+    const dataCallProxy = polygonZkEVMBridgeFactory.interface.encodeFunctionData("initialize(uint32,address,uint32,address,address,bytes)", [
```

Comment:
:thinking: it was wrong before ?

Reply:
No, it's just that before there was only one initialize function and now there are 2 (the new one is to set the proxiedTokensManager) and it create the ambiguity

Reply:
Oks! thansk for the explanation. Therefore, before it was automatically routed I guess by ethers, even the selector did not match.